crates/openid4vci/src/access_token/mod.rs

```diff
@@ -58,7 +60,18 @@ pub struct AccessTokenSuccessResponse {
 
     /// The minimum amount of time in seconds that the client SHOULD wait between polling requests to the
     /// Token Endpoint in the Pre-Authorized Code Flow. If no value is provided, clients MUST use 5 as the default.
-    pub interval: Option<u64>,
+    pub interval: Option<u32>,
+}
+
+impl Validatable for AccessTokenSuccessResponse {
+    fn validate(&self) -> Result<(), ValidationError> {
+        if self.c_nonce.is_none() && self.c_nonce_expires_in.is_some() {
```

Comment:
What is c_nonce is some, but c_nonce_expires_in is none? Is that allowed?

Reply:
The specification does not specify whether it is required when `c_nonce` is supplied (https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-6.2). So I assume it is allowed for a `c_nonce` which does not expire.